src/views/standalone/system/UpdateView.vue

```diff
@@ -0,0 +1,325 @@
+
+const scheduleDate = computed(() =>
+  systemUpdateData.value?.scheduledAt && systemUpdateData.value.scheduledAt != -1
+    ? new Date(systemUpdateData.value.scheduledAt * 1000)
+    : null
+)
+
+function cleanError() {
+  error.value.notificationTitle = ''
+  error.value.notificationDescription = ''
+  error.value.notificationDetails = ''
+}
+
+async function fetchUpdatesStatus() {
+  cleanError()
+  loading.value = true
+  try {
+    systemUpdateData.value = (await ubusCall('ns.update', 'check-system-update')).data
+    const lastPackageUpdateCheckResponse = (
+      await ubusCall('ns.update', 'get-package-updates-last-check')
+    ).data
+    if (lastPackageUpdateCheckResponse.lastCheck > 0)
+      lastPackageUpdateCheck.value = new Date(lastPackageUpdateCheckResponse.lastCheck * 1000)
+    loading.value = false
+  } catch (err: any) {
+    error.value.notificationTitle = t('error.cannot_retrieve_updates_status')
+    error.value.notificationDescription = t(getAxiosErrorMessage(err))
+    error.value.notificationDetails = err.toString()
+  }
+}
+
+async function checkPackageUpdates() {
+  cleanError()
+  try {
+    isCheckingPackageUpdates.value = true
+    noPackageUpdatesAvailable.value = false
+    packageUpdates.value = (await ubusCall('ns.update', 'check-package-updates')).data.updates
+
+    if (packageUpdates.value.length == 0) noPackageUpdatesAvailable.value = true
+  } catch (err: any) {
+    error.value.notificationTitle = t('error.cannot_retrieve_package_updates')
+    error.value.notificationDescription = t(getAxiosErrorMessage(err))
+    error.value.notificationDetails = err.toString()
+  } finally {
+    isCheckingPackageUpdates.value = false
+  }
+}
+
+async function cancelSchedule() {
+  try {
+    isCancellingSchedule.value = true
+    await ubusCall('ns.update', 'schedule-system-update', { scheduleAt: -1 })
+    closeConfirmCancelScheduleModal()
+    await fetchUpdatesStatus()
+  } catch (err: any) {
+    cancelScheduleError.value.notificationDescription = t(getAxiosErrorMessage(err))
+    cancelScheduleError.value.notificationDetails = err.toString()
+  } finally {
+    isCancellingSchedule.value = false
+  }
+}
+
+function closeConfirmCancelScheduleModal() {
+  cancelScheduleError.value.notificationDescription = ''
+  cancelScheduleError.value.notificationDetails = ''
+  showConfirmCancelScheduleDrawer.value = false
+}
+
+function showEditScheduleDrawer() {
+  showScheduleUpdateDrawer.value = true
+}
+
+onMounted(() => {
+  fetchUpdatesStatus()
+})
+</script>
+
+<template>
+  <NeTitle>{{ t('standalone.update.title') }}</NeTitle>
+  <NeInlineNotification
+    v-if="error.notificationTitle"
+    :title="error.notificationTitle"
+    :description="error.notificationDescription"
+    class="my-4"
+    kind="error"
+    ><template #details v-if="error.notificationDetails">
+      {{ error.notificationDetails }}
+    </template></NeInlineNotification
+  >
+  <NeSkeleton v-if="loading" :lines="10" />
```

Comment:
Minor skeleton improvement: can we show the title and descriptions of the two sections ("Bug & security fixes" and "System update") while loading data? The skeleton can appear only at the right column of the page